webapp/src/features/Dashboard/SubscriptionTable.js

```diff
@@ -13,8 +13,18 @@ import Table from '../../widgets/Table'
 import './Dashboard.css'
 import Icons from '../../assets/icons'
 import classNames from 'classnames'
-
+import { useSelector, useDispatch } from 'react-redux'
+import { selectAllSubscriptions } from './dashboard.selectors'
+import BusyIndicator from '../../widgets/busyIndicator'
+import {
+	setSelectedSearchType,
+	setSelectedSynchronize,
+	setSelectedSearchValue,
+} from './dashboard.slice'
 const SubscriptionTable = () => {
+	const [switchToggle, setSwitchToggle] = useState(false)
```

Comment:
what are we toggling

Reply:
we are toggling the outdatedSubscription data.